go-controller/pkg/ovn/pods.go

```diff
@@ -42,6 +42,51 @@ func (oc *Controller) syncPods(pods []interface{}) {
 				logrus.Errorf("Error in deleting pod's logical port in ovn - %s (%v)",
 					string(out), err)
 			}
+			oc.deletePodAcls(existingPort)
```

Comment:
I think this causes a race condition. Because, for each policy, we try to delete ACLs when a pod gets deleted we may see many errors in logs, when find succeeds in policy code, but delete fails.

Reply:
This is called within syncPods when checking each logical port. I am sure if this will also cause any race condition.